.github/actions/deploy-kontain/action.yaml

```diff
@@ -0,0 +1,32 @@
+name: Deploy Kontain
+description: Generate and apply Kontain deployment
+
+runs:
+  using: "composite"
+  steps:
+    - name: Build kkm and kkm_test
+      run: |
+        function wait_for_pod() {
+          set +e
+          # wait for pod to complete initialization
+          while [ -z "$status" ]
+          do 
+                  echo -n "."
+                  sleep 1s
+                  log=$(kubectl logs -l app=kontain-init -n kube-system 2>/dev/null)
+                  status="$(kubectl logs -l app=kontain-init -n kube-system 2>/dev/null | grep 'kontain-init completed')"
```

Comment:
this is cool, but we probably need to make sure we have this string there in all scenarios.

Reply:
we do, because it comes from commonly executed script. If it is not there - there was an issue